cli/tools/test_runner.rs

```diff
@@ -532,6 +535,10 @@ pub async fn run_tests(
 
         reporter.visit_event(event);
 
+        if bail > 0 && failed > bail {
```

Comment:
Should this be `failed >= bail`?

Reply:
I think that number of failing test must be greater thant `0`.

Reply:
So the description says "Exit the test suite immediately upon n number of failing test suite". So, you would want this to exit when failed == bail, correct?

Anyway, I just doing a drive by review. Would be good to get consensus on this feature first.